src/lib.rs

```diff
@@ -0,0 +1,93 @@
+use lazy_static::lazy_static;
+use std::collections::HashSet;
+
+use guest::prelude::*;
+use kubewarden_policy_sdk::wapc_guest as guest;
+
+use k8s_openapi::api::core::v1 as apicore;
+
+extern crate kubewarden_policy_sdk as kubewarden;
+use kubewarden::{logging, protocol_version_guest, request::ValidationRequest, validate_settings};
+
+mod settings;
+use settings::Settings;
+
+use slog::{error, o, warn, Logger};
```

Comment:
nit, rust idiomatic code expects the imports to be arranged in this way:

1. std library
2. 3rd party libraries
3. internal crate

Each block is separated by an empty line

Reply:
Interesting. I tough that my editor was ordering this to me. Thanks for pointing it out!